sphinxsearch/fields.py

```diff
@@ -35,8 +36,15 @@ def from_db_value(self, value, expression, connection, context):
         return datetime.datetime.fromtimestamp(value)
 
 
-class SphinxMultiField(models.IntegerField):
+class SphinxIntegerField(models.IntegerField):
```

Comment:
Please, fix README separately - need to notify users to use SphinxIntegerField instead of IntegerField.